DemoApp/DemoApp/Views/TMButtonPage.xaml

```diff
@@ -1,127 +1,195 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-             xmlns:Modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
-             x:Class="DemoApp.TMButtonPage"
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:Modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+             xmlns:controls="clr-namespace:Trimble.Modus.Components.Controls.Button;assembly=Trimble.Modus.Components"

```

Comment:
In the demo app, can we add one radio button with title 'Show icon' with the following options?
'Left', 'Right' 'Both'

Reply:
Added options to switch the image positons